src/test/java/com/sap/cloudfoundry/client/facade/CloudControllerClientIntegrationTest.java

```diff
@@ -0,0 +1,125 @@
+package com.sap.cloudfoundry.client.facade;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.jupiter.api.Assertions.fail;
+
+import java.io.IOException;
+import java.net.MalformedURLException;
+import java.net.URI;
+import java.net.URL;
+import java.util.concurrent.locks.Lock;
+import java.util.concurrent.locks.ReentrantLock;
+
+import org.apache.commons.codec.binary.StringUtils;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.http.HttpStatus;
+
+import com.sap.cloudfoundry.client.facade.domain.CloudSpace;
+
+class CloudControllerClientIntegrationTest {
+
+    private static CloudControllerClient clientWihtoutTarget;
+    private static CloudControllerClient client;
+
+    @BeforeAll
+    static void login() throws MalformedURLException {
+        ITVariable.VARIABLES.stream()
+                            .forEach(variable -> printVariable(variable));
+        CloudCredentials credentials = new CloudCredentials(ITVariable.USER_EMAIL.getValue(), ITVariable.USER_PASSWORD.getValue());
+        URL apiUrl = null;
+        apiUrl = URI.create(ITVariable.CF_API.getValue())
+                    .toURL();
+        clientWihtoutTarget = new CloudControllerClientImpl(apiUrl, credentials);
+        CloudSpace target = clientWihtoutTarget.getSpace(ITVariable.ORG.getValue(), ITVariable.SPACE.getValue());
+        client = new CloudControllerClientImpl(apiUrl, credentials, target, true);
+
+    }
+
+    // Make sure tests are not running in parallel
+    private Lock sequential = new ReentrantLock();
```

Comment:
Is there a possibility to configure the sequential order from in pom.xml plugin?

Reply:
yes, but I think it will not guarantee anything when we run tests from IDE